ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -247,6 +290,12 @@ export default defineComponent({
           default:
             return '(Not Entered)'
         }
+      }),
+      hasManualEntries: computed((): boolean => {
+        const location = getMhrRegistrationLocation.value
+        return !!location.lot || !!location.parcel || !!location.block || !!location.districtLot || !!location.partOf ||
```

Comment:
I was thinking if we could optimize here `Object.values(location).every(val => !!val)`

Reply:
Totally could if every property stored on `location` in this instance was to be considered as part of this computed condition, however it's only a select few unfortunately.

Reply:
I see, yea I didn't check if those were all of the props.. makes sense, thanks!